test-harness/specs/validate-body-params/form-dont-allow-empty-values.oas2.txt.unsupported

```diff
@@ -0,0 +1,30 @@
+====test====
+Set allowEmptyValue to false and try to send an empty key, should fail.
+
+TO BE FIXED BY: https://github.com/stoplightio/prism/issues/499
+====spec====
+swagger: '2.0'
+paths:
+  /path:
+    post:
+      consumes:
+        - application/x-www-form-urlencoded
+      responses:
+        200:
+          schema:
+            type: string
+      parameters:
+        - in: formData
+          type: string
+          name: id
+          allowEmptyValue: false
```

Comment:
Why didn't we fix this in https://github.com/stoplightio/http-spec/pull/9?

Reply:
I don't know unfortunately. I hoped it would get fixed but it didn't there must be some other quirk here but I decided not to follow up for now. I created a bug report IIRC so that I or someone else can look at it later.

Reply:
Yeah do not worry — it is effectively better to handle it as a separate task.